packages/ui/src/components/checkbox.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox'
+import { cn } from '@utils/cn'
+
+import { Icon } from './icon'
+import { Label } from './label'
+
+interface CheckboxProps extends React.ComponentPropsWithoutRef<typeof CheckboxPrimitive.Root> {
```

Comment:
```suggestion
interface CheckboxProps extends ComponentPropsWithoutRef<typeof CheckboxPrimitive.Root> {
```

let's just import from react

```tsx
import { ComponentPropsWithoutRef } from 'react'
```